lib/src/model.dart

```diff
@@ -1028,7 +1028,8 @@ class Field extends ModelElement {
     if (_field.node == null) return null;
     var v = _field.node.toSource();
     if (v == null) return null;
-    return v.substring(v.indexOf('=') + 1, v.length).trim();
+    var string = v.substring(v.indexOf('= ') + 2, v.length);
```

Comment:
Not all constants have a space between the `=`. I think we need to roll this line back.